src/Core/Geographic/Coordinates.js

```diff
@@ -0,0 +1,170 @@
+/**
+ * Generated On: 2015-10-5
+ * Class: Coordinates
+ * Description: Coordonnées cartographiques
+ */
+
+import * as THREE from 'three';
+import mE from '../Math/MathExtended';
+import Ellipsoid from '../Math/Ellipsoid';
+
+export function ellipsoidSizes() {
```

Comment:
it's better name : `ellipsoidWGS84Rayons`
and x, y and z are rayons on WGS84 ellipsoid